northstar/src/npk/npk.rs

```diff
@@ -31,6 +31,7 @@ const UNSQUASHFS: &str = "unsquashfs";
 
 // File name and directory components
 const FS_IMG_NAME: &str = "fs.img";
+const METADATA_NAME: &str = "metadata.yaml";
```

Comment:
Are we going to store more than a comment in metadata? Now it sounds like we store all metadata in its own file but we also have a manifest which contains metadata.

Reply:
The metadata just contains the version of manifest. That's why we decided to store it in the comment field.
Unfortunately reading the comment field is very expensive because a zip implementation has to seek for the end of the comment manualy.